lib/avatar_glow.dart

```diff
@@ -5,27 +5,32 @@ import 'dart:async';
 import 'package:flutter/material.dart';
 
 class AvatarGlow extends StatefulWidget {
-  final bool repeat;
-  final Duration duration;
+  final Widget child;
   final double endRadius;
+  final BoxShape shape;
+  final Duration duration;
+  final bool repeat;
+  final bool isAnimating;
```

Comment:
The field name can be changed to something more meaningful, like a normal `final bool animate`. Just my views!

Reply:
@Nolence Btw do we really need this  `animate` field? As one will only use this package when he wants to show animation on his avatar. If this field is intended for **play/pausing** the animation, then I think the name should be changed again to something more meaningful.

Reply:
A [previous issue](https://github.com/apgapg/avatar_glow/issues/1) wanted this feature and I currently use it for drawing attention to `dragTargets` that are willing to take an item (by toggling the `GlowAvatar` around them) so I don't think users will just use this package for avatars

<img width="361" alt="Screen Shot 2019-11-12 at 11 33 11 AM" src="https://user-images.githubusercontent.com/29642168/68699417-def28f80-0537-11ea-8524-7e696e466cf7.png">

I don't have a preference on naming so long as others can understand the field. I can also add doc strings to all the fields in case we need more accessibility

Reply:
Ahh, Got your point.  :+1:

Reply:
@apgapg we can merge the changes now.

Reply:
Yes sure. LGTM